app/(api)/auth/user/GET.tsx

```diff
@@ -0,0 +1,53 @@
+'use server';
```

Comment:
same as before, check every line of code you write to see if it's necessary. If you're unsure, just ask me.